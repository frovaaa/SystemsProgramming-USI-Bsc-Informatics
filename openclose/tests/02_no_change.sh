#!/bin/bash
exec "$1" <<EOF
this text should be copied verbatim to the output, since
there are no open/close pairs in it.
EOF
