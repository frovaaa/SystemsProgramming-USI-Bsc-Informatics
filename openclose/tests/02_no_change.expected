this text should be copied verbatim to the output, since
there are no open/close pairs in it.
